### PR TITLE
removing v1 embedding to fix compile on flutter 3.27

### PR DESCRIPTION
**Pull request by abdelaziz-mahdy**
fixing this https://github.com/miguelpruivo/flutter_file_picker/issues/1643

### CODE REVIEW

**Review by ivailotsvetkov — APPROVED** [review]

File: CHANGELOG.md
Comment (millerf) [inline CHANGELOG.md:1]: I believe you should Add an new version instead of removing this one, no?

Comment (abdelaziz-mahdy) [reply CHANGELOG.md:1]: I don't know what you mean, I did add a new version? 


If you are talking about v1 embedding it has been deprecated for.a  while Now, it should be removed anyway

Comment (husainazkas) [reply CHANGELOG.md:1]: I think I know what he mean, you didn't add specific new version on this changelog like the 8.1.5 one. So better you add it as 8.1.6 and the package owner should merge this pr then publish to pub.dev

Comment (deakjahn) [reply CHANGELOG.md:1]: I'm not a maintainer but the changelog looks just fine to me.

**Review by InvertedX — APPROVED** [review]

File: CHANGELOG.md
Comment (InvertedX) [reply CHANGELOG.md:1]: I've tested this on my setup, and everything works as expected. Great job! @miguelpruivo

Comment (millerf) [reply CHANGELOG.md:1]: > I don't know what you mean, I did add a new version?

Sorry, my bad... I did not have my coffee yet when I reviewed :)

Comment (MrCsabaToth) [reply CHANGELOG.md:1]: Bumping version is good until another PR with version bump is merged. It is up to the maintainer if he requires version bump as part of the PR. Personally I do the version bump in the release branch dedicated the version release. I know that for example Flutter community plugins require version bump (I've been through it and got hindered several times due to other PRs with version merging and having to resolve conflict and rebase).
Just merge this PR (🙏🏻 🙏🏻 🙏🏻) with or without version FFS.

**Review by MrCsabaToth — APPROVED** [review]
Just let it merge please @miguelpruivo

**Comment by reidbaker** [comment]
The bug has a huge number of comments so I am adding some context here. 

My teammate on flutter removed v1 embeddings in this pr https://github.com/miguelpruivo/flutter_file_picker/pull/1527 fixing the issue we filed https://github.com/miguelpruivo/flutter_file_picker/issues/1526. 

Is appears that in pull request [#1453](https://github.com/miguelpruivo/flutter_file_picker/pull/1453/files#diff-a28f06c7f47e384adf16f883f6bd99eae4cabc9c10f532d8fe0eee8c33a9ce00)  the lines we removed were added back in. 

5 days ago flutter released 3.27 which was the first stable release to not have the v1 embedding code so any user with this plugin and 3.27 would fail to compile. 

At first glance this pr appears to address the regression.

**Review by reidbaker — APPROVED** [review]

**Review by navaronbracke — APPROVED** [review]
LGTM! Thanks for fixing this regression!

**Pull request merged**
